test/test/helpers/use-fake-dom/index.js

```diff
@@ -25,8 +25,8 @@ export default function domWrapper( markup, features ) {
 
 domWrapper.withContainer = function withContainer() {
```

Comment:
Yes, I'll re-export `withContainer` and `getContainer`. Unfortunately we need to keep customised `domWrapper` because of `page` cache cleanup. Unless we find a better way to handle that :)